native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/sql-native/reflect-config.json

```diff
@@ -0,0 +1,33 @@
+[
+{
+  "name":"[Lcom.zaxxer.hikari.util.ConcurrentBag$IConcurrentBagEntry;"
```

Comment:
Lets format this json file

Reply:
Fixed in [7655a3a](https://github.com/ballerina-platform/module-ballerina-sql/pull/563/commits/7655a3a7e3e4de9e864c1cdc93eeedaad7952a61)